### PR TITLE
Update release tags.

### PR DESCRIPTION
**Pull request by bbastings**

### CODE REVIEW

File: core/frontend/src/Viewport.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ export enum DepthPointSource {
   Map,
 }
 
-/** @alpha Options to control behavior of [[Viewport.pickDepthPoint]]. */
+/** @public Options to control behavior of [[Viewport.pickDepthPoint]]. */
```
Comment (calebmshafer) [inline core/frontend/src/Viewport.ts:107]: Not sure if this formatting will show up correctly in the docs.  You will probably need to move the actual description to the first line and have the release tag on a second line.

File: core/frontend/src/tools/ClipViewTool.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ import { ToolAssistance, ToolAssistanceImage, ToolAssistanceInputMethod, ToolAss
 
 // cSpell:ignore geti
 
-/** @alpha An object that can react to a view's clip being changed by tools or modify handles. */
+/** @public An object that can react to a view's clip being changed by tools or modify handles. */
 export interface ViewClipEventHandler {
   selectOnCreate(): boolean; // Add newly created clip geometry to selection set and show modify controls.
```
Comment (pmconne) [inline core/frontend/src/tools/ClipViewTool.ts:36]: Turn the `//` comments into `/** doc comments */`.

File: core/frontend/src/tools/ClipViewTool.ts [hunk 3]
```diff
@@ -60,7 +60,7 @@ export interface DrawClipOptions {
   id?: string;
 }
 
-/** @alpha A tool to define a clip volume for a view */
+/** @public A tool to define a clip volume for a view */
 export class ViewClipTool extends PrimitiveTool {
   constructor(protected _clipEventHandler?: ViewClipEventHandler) { super(); }
```
Comment (pmconne) [inline core/frontend/src/tools/ClipViewTool.ts:70]: Are protected members like `_orientationName` supposed to be `@public`?

File: core/frontend/src/tools/ClipViewTool.ts [hunk 4]
```diff
@@ -60,7 +60,7 @@ export interface DrawClipOptions {
   id?: string;
 }
 
-/** @alpha A tool to define a clip volume for a view */
+/** @public A tool to define a clip volume for a view */
```
Comment (pmconne) [inline core/frontend/src/tools/ClipViewTool.ts]: Undocumented overrides of base class methods like `onResetButtonUp` clutter the documentation and are not really part of the API - can you mark them as internal?

File: core/frontend/src/tools/MeasureTool.ts [hunk 5]
```diff
@@ -152,7 +152,7 @@ function adjustPoint(ev: BeButtonEvent, segments?: Array<Segment>, locations?: A
   return ev.point;
 }
 
-/** @alpha */
+/** @public */
```
Comment (pmconne) [inline core/frontend/src/tools/MeasureTool.ts:155]: Can you add some documentation to these tools, like a description of what they do?

**Review by pmconne — CHANGES_REQUESTED** [review]

File: core/frontend/src/tools/ClipViewTool.ts (hunk 4 above)
Comment (bbastings) [reply core/frontend/src/tools/ClipViewTool.ts]: Should I just make every method @internal? I'd just like people to be able to install these tools, sub-class them to extend them isn't the goal...

Comment (pmconne) [reply core/frontend/src/tools/ClipViewTool.ts]: Probably. If somebody does want to subclass them, they'd better be looking at the actual code anyway, not the API docs.
I've been marking things like this as `@internal override` so I remember why they were made internal.

File: core/frontend/src/tools/ClipViewTool.ts (hunk 2 above)
Comment (bbastings) [reply core/frontend/src/tools/ClipViewTool.ts]: done

File: core/frontend/src/tools/MeasureTool.ts (hunk 5 above)
Comment (bbastings) [reply core/frontend/src/tools/MeasureTool.ts]: done

File: core/frontend/src/Viewport.ts (hunk 1 above)
Comment (bbastings) [reply core/frontend/src/Viewport.ts]: Changed just to be safe

File: core/frontend/src/tools/ClipViewTool.ts (hunk 4 above)
Comment (bbastings) [reply core/frontend/src/tools/ClipViewTool.ts]: done

File: core/frontend/src/tools/ClipViewTool.ts (hunk 3 above)
Comment (bbastings) [reply core/frontend/src/tools/ClipViewTool.ts:77]: done (to keep these out of the docs)

**Review by pmconne — APPROVED** [review]

**Pull request merged**